Cache/TreeCache.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace Becklyn\RouteTreeBundle\Cache;
+
+use Becklyn\RouteTreeBundle\Tree\Node;
+use Symfony\Component\HttpKernel\HttpKernel;
+use Symfony\Component\HttpKernel\KernelInterface;
+
+
+/**
+ *
+ */
+class TreeCache
+{
+    /**
+     * @var bool
+     */
+    private $isDebug;
+
+    /**
+     * @var string
+     */
+    private $filePath;
+
+
+    /**
+     * @param KernelInterface $kernel
+     */
+    public function __construct (KernelInterface $kernel)
```

Comment:
I am not sure how can just extract the one method result (`getCacheDir()`) into it's own parameter.